KarhooUISDK/Screens/QuoteList/QuoteListErrorView.swift

```diff
@@ -7,36 +7,29 @@ import Foundation
 import KarhooSDK
 import UIKit
 
-protocol QuoteListErrorViewDelegate: AnyObject {
+@objc protocol QuoteListErrorViewDelegate: AnyObject {
+    func showNoCoverageEmail()
 }
 
-final class QuoteListErrorView: UIView {
-
-    // MARK: - Nested types
-    
-    private enum Constants {
-        static let imageViewSize = CGSize(width: 120, height: 120)
-        static let titleFontSize = CGFloat(18)
-        static let messageFontSize = CGFloat(14)
-    }
-
-    // MARK: - Views
+final class QuoteListErrorView: UIView, UITextViewDelegate {
 
     private lazy var titleLabel = UILabel().then {
-        $0.font = KarhooUI.fonts.getSemiboldFont(withSize: Constants.titleFontSize)
+        $0.font = KarhooUI.fonts.subtitleSemibold()
         $0.textColor = KarhooUI.colors.text
         $0.textAlignment = .center
         $0.numberOfLines = 0
         $0.text = viewModel.title
     }
 
-    private lazy var descriptionLabel = UILabel().then {
-        $0.font = KarhooUI.fonts.getRegularFont(withSize: Constants.messageFontSize)
+    private lazy var descriptionLabel = UITextView().then {
```

Comment:
Change object identifier to contain proper type name. For example, `descriptionTextView`.